app/views/conferences/_booths.html.haml

```diff
@@ -0,0 +1,26 @@
+= content_for :splash_nav do
+  %li
+    %a.smoothscroll{ href: '#booths' } Booths
+
+.container
+  .row
+    .col-md-12.text-center
+      %h2 Booths
+  - i = 0;
+  - @conference.booths.confirmed.each_slice(3) do |slice|
+    .row.row-centered
+      - slice.each do |booth|
+        .col-md-4.col-sm-4.col-centered.col-top
+          .thumbnail
+            - if booth.logo_link
+              = link_to booth.website_url, class: 'thumbnail' do
+                = image_tag booth.picture.large.url
+            .caption
+              %h3.text-center
+                = booth.title
+              %p.text-center.text-muted
+                = link_to "#show_descrition_#{i}", "data-toggle"=>"collapse" do
+                  learn more
+              .collapse{ id: "show_descrition_#{i}" }
+                = markdown(booth.description)
+        - i = i + 1
```

Comment:
if we don't have slice we can still have 3 booths per row, but when there is only one booth the output is like this and I personally I dislike it
![20773345_1819433134741114_368000540_o](https://user-images.githubusercontent.com/24506693/29132493-0ab40038-7d39-11e7-863e-4cafbc06d454.png)